### PR TITLE
[zh] Sync web page for crictl

### PR DESCRIPTION
**Pull request by chenxuc**
This is part of the work in the following umbrella issue:
[zh] Umbrella issue: pages out of sync in tasks section #26178

CRICTL (L)

 content/zh/docs/tasks/debug-application-cluster/crictl.md

### CODE REVIEW

**Comment by ydFu** [comment]
@chenxuc 

In the PR description, please add Related: #26178

> [zh] Umbrella issue: pages out of sync in tasks section

to
 
> [zh] Umbrella issue: pages out of sync in tasks section(#26178)

Thank for your contribution.

**Comment by chenxuc** [comment]
> @chenxuc
> 
> In the PR description, please add Related: #26178
> 
> > [zh] Umbrella issue: pages out of sync in tasks section
> 
> to
> 
> > [zh] Umbrella issue: pages out of sync in tasks section(#26178)
> 
> Thank for your contribution.

Added the related issue number. Thanks.

File: content/zh/docs/tasks/debug-application-cluster/crictl.md [hunk 1]
```diff
@@ -454,3 +454,124 @@ for more information.
...
+`inspect` | `inspect`, `inspecti` | Return low-level information on a container, image or task |  
+`logs` | `logs` | Fetch the logs of a container | `--details`
+`ps` | `ps` | List containers |  
+`stats` | `stats` | Display a live stream of container(s) resource usage statistics | Column: NET/BLOCK I/O, PIDs
+`version` | `version` | Show the runtime (Docker, ContainerD, or others) version information |  
+{{< /table >}}
+-->
+docker cli | crictl | 描述 | 不支持的功能
+-- | -- | -- | --
+`attach` | `attach` | 连接到一个运行的容器 | `--detach-keys`, `--sig-proxy`
```
Comment (tengqm) [inline content/zh/docs/tasks/debug-application-cluster/crictl.md:502]:
```suggestion
`attach` | `attach` | 连接到一个运行中的容器 | `--detach-keys`, `--sig-proxy`
```

File: content/zh/docs/tasks/debug-application-cluster/crictl.md [hunk 2]
```diff
@@ -454,3 +454,124 @@ for more information.
...
+`exec` | `exec` | 在运行的容器里运行一个命令 | `--privileged`, `--user`, `--detach-keys`
```
Comment (tengqm) [inline content/zh/docs/tasks/debug-application-cluster/crictl.md:503]:
```suggestion
`exec` | `exec` | 在运行中的容器里运行一个命令 | `--privileged`, `--user`, `--detach-keys`
```

File: content/zh/docs/tasks/debug-application-cluster/crictl.md [hunk 3]
```diff
@@ -454,3 +454,124 @@ for more information.
...
+`run` | `run` | Run a command in a new container |  
+`start` | `start` | Start one or more stopped containers | `--detach-keys`
+`stop` | `stop` | Stop one or more running containers |  
+`update` | `update` | Update configuration of one or more containers | `--restart`, `--blkio-weight` and some other resource limit not supported by CRI.
+{{< /table >}}
+-->
+docker cli | crictl | 描述 | 不支持的功能
+-- | -- | -- | --
+`create` | `create` | 创建一个新的容器 |  
+`kill` | `stop` (timeout = 0) | 杀死一个或多个正在运行的容器 | `--signal`
```
Comment (tengqm) [inline content/zh/docs/tasks/debug-application-cluster/crictl.md:538]:
```suggestion
`kill` | `stop` (timeout=0) | 杀死一个或多个正在运行的容器 | `--signal`
```

File: content/zh/docs/tasks/debug-application-cluster/crictl.md [hunk 4]
```diff
@@ -454,3 +454,124 @@ for more information.
...
+`stop` | `stop` | 停止一个或多个正运行的容器 |  
+`update` | `update` | 更新一个或多个容器的配置 | CRI 不支持 `--restart`、`--blkio-weight` 以及一些其他的资源限制选项。
+{{< /table >}}
+
+<!--
+### Supported only in crictl
+
+{{< table caption="mapping from docker cli to crictl - supported only in crictl" >}}
+-->
+### 仅支持 crictl
```
Comment (tengqm) [inline content/zh/docs/tasks/debug-application-cluster/crictl.md:553]:
```suggestion
### 仅 crictl 支持
```

File: content/zh/docs/tasks/debug-application-cluster/crictl.md [hunk 5]
```diff
@@ -454,3 +454,124 @@ for more information.
...
+
+{{< table caption="Docker CLI 和 crictl 的映射 - 仅支持 crictl" >}}
```
Comment (tengqm) [inline content/zh/docs/tasks/debug-application-cluster/crictl.md:555]:
```suggestion
{{< table caption="Docker CLI 和 crictl 的映射 - 仅 crictl 支持" >}}
```

**Comment by chenxuc** [comment]
@tengqm Thanks for the review. I've updated the commit.

File: content/zh/docs/tasks/debug-application-cluster/crictl.md [hunk 6]
```diff
@@ -454,3 +454,124 @@ for more information.
...
+<!--
+## Mapping from docker cli to crictl
+-->
+## Docker CLI 和 crictl 的映射
+
+<!--
+The exact versions for below mapping table are for docker cli v1.40 and crictl v1.19.0. Please note that the list is not exhaustive. For example, it doesn't include experimental commands of docker cli.
+-->
+以下的映射表格只适用于 Docker CLI v1.40 和 crictl v1.19.0 版本。
+请注意该表格并不祥尽。例如，其中不包含 Docker CLI 的实验性命令。
```
Comment (tengqm) [inline content/zh/docs/tasks/debug-application-cluster/crictl.md:466]:
```suggestion
请注意该表格并不详尽。例如，其中不包含 Docker CLI 的实验性命令。
```

**Comment by chenxuc** [comment]
@tengqm Thanks for the catch. I've fixed it again. 
BTW, is there any tool you use to find such hidden typo? Thanks.

**Comment by tengqm** [comment]
@chenxuc My tool is called "eyes".
/lgtm

**Pull request merged**